oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -535,6 +539,10 @@ health-checker:
   default:
     checkIntervalSeconds: ${SW_HEALTH_CHECKER_INTERVAL_SECONDS:5}
 
+debugging-query:
+  selector: ${SW_DEBUGGING_QUERY:default}
```

Comment:
For such a plugin for debugging purpose, I would say users only want to enable it when they are debugging something, and should be turned off after debugging, I don't think there is any reason to turn it on by default

Reply:
Changing the value means rebooting, I don't want that happen.

Reply:
> Changing the value means rebooting, I don't want that happen.

I think it's a common sense that debugging means `do adjust codes or configs; reboot; until it works as expected`, expose these configs might be risky so if anyone missed a keyword (like https://github.com/apache/skywalking/pull/11579/files#r1402079849), they have to reboot their production env.